pkgs/os-specific/linux/tomb/default.nix

```diff
@@ -1,29 +1,42 @@
 { stdenv
 , lib
 , fetchFromGitHub
+, substituteAll
 , makeWrapper
+, getent
 , gettext
 , zsh
 , pinentry
 , cryptsetup
 , gnupg
+, libargon2
+, lsof
 , util-linux
 , e2fsprogs
-, sudo
+, testers
+, nix-update-script
+, sudoPath ? "/run/wrappers/bin/sudo"
```

Comment:
A problem with this approach is that it does not work outside of nixos.

Reply:
@pbsds Hello. Thanks for the review.
Do you have any ideas about the best way to solve this problem?

I don't have any other options apart from allowing the user to override the `sudoPath` attribute according to their system.

An another option could be also to remove this `sudoPath` attribute and the sudo buildInput, but this would make installation of the `tomb` package conditional on the presence of the sudo binary in its PATH.

Do you have any idea?

Reply:
Perhaps a `pkgs.writeShellScript` replacement? A simple script that checks for the existence of the nixos sudo path then tries to use it, and otherwise falls back to the sudo in PATH.

Reply:
Why not always use PATH?

Reply:
Okay thanks for the review, I changed my PR to use the sudo binary in the PATH instead.